sw/device/lib/meson.build

```diff
@@ -22,6 +21,8 @@ sw_lib_uart = declare_dependency(
   )
 )
 
+subdir('testing')
```

Comment:
Is there a specific reason for moving this down here?

Reply:
Yes, it's because test_main.c (currently) depends on uart.c.

Reply:
I see, thanks. Perhaps worth a comment.